tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,87 @@
+import { getPageTitle, startAtHomepage } from "../helper";
+
+import {
+  hasCreateQuestionnaireButton,
+  clickCreateQuestionnaire
+} from "../pages/AuthorHome.page";
+import {
+  onCreateQuestionnairePage,
+  enterQuestionnaireDetails,
+  clickCreateButton
+} from "../pages/CreateQuestionnaire.page";
+import {
+  onDesignQuestionnairePage,
+  setSectionTitle,
+  setQuestionTitle,
+  clickAddAnswer,
+  selectTextFieldAnswer,
+  setAnswerTitle,
+  getFirstSectionTitle,
+  clickAddPage,
+  clickPreview
+} from "../pages/DesignQuestionnaire.page";
+
+describe("eQ Services Smoke Test", () => {
+    beforeAll(() => {
+      startAtHomepage();
+      clickCreateQuestionnaire();
+      enterQuestionnaireDetails(
+        "title",
+        "description",
+        "surveyId",
+        "default",
+        "StatisticsOfTradeAct",
+        true
+      );
+      clickCreateButton();
+    });
+
+    it("should update navigation title when section title changed", () => {
+      expect(getFirstSectionTitle()).toEqual("Section Title");
+
+
+      // Adds section title
+      setSectionTitle("eQ Services Smoke Test");
+      browser.keys("Tab");
+      browser.waitForExist("h3[class*='SectionTitle']")
+      expect(getFirstSectionTitle()).toEqual("eQ Services Smoke Test");
+
+      // Adds first question
+      setQuestionTitle("Test Question 1");
+      browser.keys("Tab");
+      clickAddAnswer();
+      browser.waitForExist("button[title='Text']")
+      selectTextFieldAnswer();
+      browser.waitForExist("#question-page-editor [id='label']")
+      setAnswerTitle("Test Answer 1");
+
+      // Adds second Question
+      clickAddPage();
+      setQuestionTitle("Test Question 2");
+      browser.keys("Tab");
+      clickAddAnswer();
+      browser.waitForExist("button[title='Text']")
+      selectTextFieldAnswer();
+      browser.waitForExist("#question-page-editor [id='label']")
+      setAnswerTitle("Test Answer 2");
+
+      // Preview and verifies the newly created questionnaire
+      clickPreview();
+      var tabIds = browser.getTabIds();
+      browser.switchTab(tabIds[1])
+      expect(getPageTitle()).toEqual("Test Question 1 - title");
+
+
+      // Edits the title and previews the survey again to assert the edited title and confirm that we are not caching data
+      browser.switchTab(tabIds[0])
+      setSectionTitle(" Edited");
+      browser.keys("Tab");
+
+      clickPreview();
+      var tabIds = browser.getTabIds();
+      browser.switchTab(tabIds[2])
+
+      expect(browser.getText("p[data-qa='block-title']")).toEqual("eQ Services Smoke Test Edited");
```

Comment:
To properly assert you have edited then you need to check the same value before and after the edit.

Reply:
addressed